docs/framework/install/on-windows-7.md

```diff
@@ -3,7 +3,6 @@ title: Install the .NET Framework on Windows 7 SP1 | Microsoft Docs
 description: Learn how to install .NET Framework on Windows 7 SP1
 author: rlander
 keywords: .NET Framework, Install
-
 ms.date: 03/28/2017
```

Comment:
these changes we're making to these topics would grant a change on the ms.date since they're significant changes to make sure the topic is up-to-date and accurate